File: github-enterprise-server/TBL-GithubEnterpriseServerEvidenceCollector-DotNet/SampleApp_GithubEnterpriseServer/Models/GitHubPullRequestReviewsResponseModel.cs
using System;

namespace SampleApp_GithubEnterpriseServer.Models
{
    /// <summary>
    /// Response model used for deserializing the JSON response from
    /// the GitHub Enterprise Server API
    /// </summary>
    public class GitHubPullRequestReviewsResponseModel
    {
        public int id { get; set; }
        public string node_id { get; set; }
        public User user { get; set; }
        public DateTime submitted_at { get; set; }
        public string author_association { get; set; }
        public string body { get; set; }
        public string state { get; set; }
    }
}

File: github-enterprise-server/TBL-GithubEnterpriseServerEvidenceCollector-Java/src/com/tugboatlogic/sample/models/GitHubPullRequestReviewsResponseModel.java
package com.tugboatlogic.sample.models;

/*
 * Response model used for deserializing the JSON response from
 * the GitHub Enterprise Server API
 */
public class GitHubPullRequestReviewsResponseModel {
	public int id;
	public String node_id;
	public User user;
	public String submitted_at;
	public String author_association;
	public String body;
	public String state;
}
